### DISCUSSION TITLE
q-select

### DISCUSSION THREAD

**Discussion by talbene** [quasarframework/quasar#11199] [General - Components / Directives / etc]
Hey.
im having issue with my q-select when i select sometimes one of the options is missing, 
i need to close and open it in order to see it:
![image](https://user-images.githubusercontent.com/10762238/140036699-f9966ac5-1358-464f-8dc8-12908ec9769c.png)

anyone encounter this issue ?

thanks.

**Comment by MilosPaunovic** [comment]
Please provide a Codepen reproduction of the problem so you could get better support.

**Comment by talbene** [comment]
![image](https://user-images.githubusercontent.com/10762238/140044190-995d5d2b-5574-423e-a13b-ea00e5da4c78.png)

**Reply by hawkeye64** [reply]
Try this:
![image](https://user-images.githubusercontent.com/10262924/140801587-2a00b22d-ba4b-47fe-90ca-7ef21d652e95.png)
